### DISCUSSION TITLE
Blokas PiSound as audio out?

### DISCUSSION THREAD

**Discussion by dcoredump** [probonopd/MiniDexed#275] [Q&A] [answered]
I just tried to use my Blokas [PiSound](https://blokas.io/pisound/) audio card (which has audio in and out as well as MIDI in and out) but it seems not to work out-of-the-box as replacement for my Hifiberry card. MIDI seems to work but I don't hear anything 😞.

So I think it's a kind of "driver" problem. Has anyone got PiSound working with circle?

TIA, Holger

**Comment by probonopd** [comment, marked as answer]
https://github.com/probonopd/MiniDexed/discussions/149#discussioncomment-2619691

Can you confirm it has a PCM5102A?

**Comment by dcoredump** [comment]
Now I had some time to check this: Yes, it is a PCM5102A. I will try to follow #149 to get it to work.

Thx!

**Reply by probonopd** [reply]
Did you get somewhere with it?

**Reply by dcoredump** [reply]
No, not really. I think to get sound out of the PiSound you only have to set the GPIOs for the output quality. But the MIDI interface is not the standard serial console. They are using a SPI interface for which the driver code has to be written...
